language/move-stdlib/src/main.rs

```diff
@@ -0,0 +1,12 @@
+use move_stdlib::utils::time_it;
+
+fn main() {
+    // Generate documentation
+    {
+        time_it("Generating stdlib documentation", || {
```

Comment:
It seems to me that by invoking the doc-gen, the compiler is automatically run so there's no need to call it separately. Please let me know if this is correct!